lib/src/test/java/io/ably/lib/test/realtime/RealtimeInitTest.java

```diff
@@ -219,4 +219,62 @@ public void println(int severity, String tag, String msg, Throwable tr) {
 			if(ably != null) ably.close();
 		}
 	}
+
+	/**
+	 * Init library with 'production' environment
+	 * Spec: RTC1e
+	 */
+	@Test
+	public void init_production_environment() {
+		try {
+			TestVars testVars = Setup.getTestVars();
+			ClientOptions opts = new ClientOptions(testVars.keys[0].keyStr);
+			opts.environment = "production";
+			AblyRealtime ably = new AblyRealtime(opts);
+			assertEquals("Unexpected host mismatch", Defaults.HOST_REALTIME, ably.options.realtimeHost);
+		} catch (AblyException e) {
+			e.printStackTrace();
+			fail("init_production_environment: Unexpected exception instantiating library");
+		}
+	}
+
+	/**
+	 * Init library with given environment
+	 * Spec: RTC1e
+	 */
+	@Test
+	public void init_given_environment() {
+		final String givenEnvironment = "staging";
+		try {
+			TestVars testVars = Setup.getTestVars();
+			ClientOptions opts = new ClientOptions(testVars.keys[0].keyStr);
+			opts.environment = givenEnvironment;
+			AblyRealtime ably = new AblyRealtime(opts);
+			assertEquals("Unexpected host mismatch", String.format("%s-%s", givenEnvironment, Defaults.HOST_REALTIME), ably.options.realtimeHost);
+		} catch (AblyException e) {
+			e.printStackTrace();
+			fail("init_given_environment: Unexpected exception instantiating library");
+		}
+	}
+
+	/**
+	 * Init library with given environment and specified host
+	 * Spec: RTC1e
+	 */
+	@Test
+	public void init_given_host_environment() {
+		final String givenEnvironment = "staging";
+		final String specifiedHost = "fake.ably.io";
+		try {
+			TestVars testVars = Setup.getTestVars();
+			ClientOptions opts = new ClientOptions(testVars.keys[0].keyStr);
+			opts.realtimeHost = specifiedHost;
+			opts.environment = givenEnvironment;
```

Comment:
So, I can't use `realtimeHost` and `environment` together for init `AblyRealtime`. 
If I want to use "env-test.realtime.host" as `realtimeHost`, I should set this host like 
`options.realtimeHost="env-test.realtime.host"`
instead 
`options.environment = "env"` 
`options.realtimeHost = "test.realtime.host"`.
`environment` name can be used only for default host endpoint.
And the same logic for `AblyRest`. Right?